src/roman_datamodels/testing/utils.py

```diff
@@ -340,20 +431,135 @@ def add_ref_common(meta):
         '2020-01-01T00:00:00.0', format='isot', scale='utc')
     meta['reftype'] = ''
 
-def mk_flat_file(outfilepath, shape=(20, 20)):
+def mk_level1_science_raw(shape=None, filepath=None):
+    """
+    Create a dummy level 1 ScienceRaw instance (or file) with arrays and valid values for attributes
+    required by the schema.
+
+    Parameters
+    ----------
+    shape
+        (optional) Shape of arrays in the model.
+
+    filepath
+        (optional) File name and path to write model to.
+
+    Returns
+    -------
+    roman_datamodels.stnode.WfiScienceRaw
+    """
+    meta = mk_common_meta()
+    wfi_science_raw = stnode.WfiScienceRaw()
+    wfi_science_raw['meta'] = meta
+
+    if not shape:
+        shape = (8, 4096, 4096)
+
+    wfi_science_raw['data'] = np.zeros(shape, dtype=np.uint16)
+
+    if filepath:
+        af = asdf.AsdfFile()
+        af.tree = {'roman': wfi_science_raw}
+        af.write_to(filepath)
+    else:
+        return wfi_science_raw
+
+
+def mk_level2_image(shape=None, filepath=None):
+    """
+    Create a dummy level 2 Image instance (or file) with arrays and valid values for attributes
+    required by the schema.
+
+    Parameters
+    ----------
+    shape
+        (optional) Shape of arrays in the model.
+
+    filepath
+        (optional) File name and path to write model to.
+
+    Returns
+    -------
+    roman_datamodels.stnode.WfiImage
+    """
+    meta = mk_common_meta()
+    wfi_image = stnode.WfiImage()
+    wfi_image['meta'] = meta
+
+    if not shape:
+        shape = (4096, 4096)
+
+    wfi_image['data'] = np.zeros(shape, dtype=np.float32)
+    wfi_image['dq'] = np.zeros(shape, dtype=np.uint32)
+    wfi_image['err'] = np.zeros(shape, dtype=np.float32)
+    wfi_image['var_poisson'] = np.zeros(shape, dtype=np.float32)
+    wfi_image['var_rnoise'] = np.zeros(shape, dtype=np.float32)
+    wfi_image['var_flat'] = np.zeros(shape, dtype=np.float32)
+    wfi_image['area'] = np.zeros(shape, dtype=np.float32)
+    wfi_image['cal_logs'] = mk_cal_logs()
+
+    if filepath:
+        af = asdf.AsdfFile()
+        af.tree = {'roman': wfi_image}
+        af.write_to(filepath)
+    else:
+        return wfi_image
+
+
+def mk_flat(shape=None, filepath=None):
+    """
+    Create a dummy Flat instance (or file) with arrays and valid values for attributes
+    required by the schema.
+
+    Parameters
+    ----------
+    shape
+        (optional) Shape of arrays in the model.
+
+    filepath
+        (optional) File name and path to write model to.
+
+    Returns
+    -------
+    roman_datamodels.stnode.FlatRef
+    """
     meta = {}
     add_ref_common(meta)
     flatref = stnode.FlatRef()
     meta['reftype'] = 'FLAT'
     flatref['meta'] = meta
+
+    if not shape:
+        shape = (4096, 4224)
+
```

Comment:
The current flat are 4096x 4096 and we should update the default to match.

Reply:
All are changed to that shape.